### PR TITLE
Feature/add puzzle

### PR DESCRIPTION
**Pull request by JRicardoT**
**What (if any) features are you implementing?**
- User can add a puzzle after filling out form completely

**What (if anything) did you refactor?**
- N/A

**Were there any issues that arose?**
- Have to figure out how to upload an image.

**Is there anything needs to be prioritized?**
- Figure out how to upload a picture and render it on the DOM.

**Next Steps?**
- Figure out how to upload a picture and render it on the DOM.

### CODE REVIEW

File: src/Context.tsx [hunk 1]
```diff
@@ -52,12 +53,22 @@ const PuzzleProvider = ({children}: IPuzzleProvider) => {
 		setLoggedIn(true)
 	}
 
+	const addPuzzle = (newPuzzle: ICleanedPuzzleObject) => {
+		console.log('newPuzzle>>>>', newPuzzle)
+		setPuzzles([...puzzles, newPuzzle])
```
Comment (ccollums) [inline src/Context.tsx:58]: this will need to be deleted once we do the POST request, yeah??

File: src/components/AddPuzzleForm.tsx [hunk 2]
```diff
@@ -1,68 +1,171 @@
...
+import { ICleanedPuzzleObject } from '../interfaces';
+import { PuzzleContext } from '../Context';
 import '../css/AddPuzzleForm.css'
 
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  props,
+  ref,
+) {
+  return <MuiAlert elevation={6} ref={ref} variant="filled" {...props} />;
```
Comment (ccollums) [inline src/components/AddPuzzleForm.tsx:14]: I love this alert! Looks so sleek and clean

**Review by ccollums — COMMENTED** [review]
This looks great! Just will need to delete the commented out code!

**Review by kbergsund — APPROVED** [review]
Code reviewed together over Zoom. Loving the MUI alert feedback!

**Pull request merged**